doc/releases/pacific.rst

```diff
@@ -2,6 +2,339 @@
 Pacific
 =======
 
+v16.2.8 Pacific   
+===============
+
+This is the eighth backport release in the Pacific series.
+
+Notable Changes
+---------------
+
+* MON/MGR: Pools can now be created with `--bulk` flag. Any pools created with `bulk`
+  will use a profile of the `pg_autoscaler` that provides more performance from the start.
+  However, any pools created without the `--bulk` flag will remain using it's old behavior
+  by default. For more details, see:
+
+  https://docs.ceph.com/en/latest/rados/operations/placement-groups/
+
+* MGR: The pg_autoscaler can now be turned `on` and `off` globally
+  with the `noautoscale` flag. By default this flag is unset and
+  the default pg_autoscale mode remains the same.
+  For more details, see:
+
+  https://docs.ceph.com/en/latest/rados/operations/placement-groups/
+
+* A health warning will now be reported if the ``require-osd-release`` flag is not
+  set to the appropriate release after a cluster upgrade.
+
+Changelog
+---------
+
+* [Revert] bluestore: set upper and lower bounds on rocksdb omap iterators (`pr#46092 <https://github.com/ceph/ceph/pull/46092>`_, Neha Ojha)
```

Comment:
Should we remove this along with the initial PR from the changelog, leaving the PR that actually implements this functionality (46096)?

Reply:
I think we could just include the PRs that have merged since the time we created this list to avoid any confusion. @yuriw could you please add the new PRs?

Reply:
Fwiw, it's already come up on the mailing list as adding confusion:

```
I've seen that the 16.2.8 was out yesterday, but I'm a little confused on :
[Revert] bluestore: set upper and lower bounds on rocksdb omap iterators (pr#46092, Neha Ojha)
bluestore: set upper and lower bounds on rocksdb omap iterators (pr#45963, Cory Snyder)

(theses two lines seems related to https://tracker.ceph.com/issues/55324).

One step forward, one step backward ?
```

I think what matters is intent. Is the changelog intended to just be a list of all PRs for consumption by developers, or a list of changes users can expect? If the latter, then I think this is more confusing.